lib/gems/pending/appliance_console/key_configuration.rb

```diff
@@ -63,6 +66,17 @@ def activate
       end
     end
 
+    def save_temp_key
+      FileUtils.mv(KEY_FILE_TEMP, KEY_FILE)
+    rescue StandardError => e
```

Comment:
You suggested was the one I originally used. But Yuri has different idea on this. He suggest to use a common parent class of the exception we may encountered. And we made sure all exceptions FileUtils.mv made is a subclass of StandardError, so I'll leave this unchanged. Thank you.